ci/rpm_build_success.sh

```diff
@@ -0,0 +1,20 @@
+# /bin/sh
```

Comment:
(lint) Tips depend on target shell and yours is unknown. Add a shebang. [SC2148]